doc/api/cli.md

```diff
@@ -1749,6 +1749,15 @@ Enables report to be generated when the process exits due to an uncaught
 exception. Useful when inspecting the JavaScript stack in conjunction with
 native stack and other runtime environment data.
 
+### `--report-exclude-network`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Exclude `header.networkInterfaces` from the diagnostic report. Default is
+`false`.
```

Comment:
```suggestion
Exclude `header.networkInterfaces` from the diagnostic report. By default
this is not set and the network interfaces are included.
```

This should be described without mentioning booleans because to unset it you don't actually pass a false value but do --no-...